src/qt/askpassphrasedialog.cpp

```diff
@@ -240,10 +234,6 @@ bool AskPassphraseDialog::eventFilter(QObject *object, QEvent *event)
                 ui->capsLabel->clear();
             }
         }
-    } else if (event->type() == QEvent::FocusIn) {
-        if (object->inherits("QLineEdit")) {
-            keyboard->setInput(object);
-        }
     }
     return false;
-}
+}
```

Comment:
Good pickup, I must have gone and removed it in the original commit but I have now put it back.

I usually do try and remember to include the new line, as you said it may not be necessary anymore but it's still standard procedure. At least in this codebase.